generator/composer.json

```diff
@@ -0,0 +1,11 @@
+{
+    "require-dev": {
+        "php" : "^7",
+        "phpunit/phpunit": "^7.4",
+        "smarty/smarty": "^3.1"
+    },
+    "autoload": {
+        "psr-4": { "Google\\": "src/Google" }
```

Comment:
One of the neat things introduced with PSR-4 is that you can have a more condensed directory structure. Changing this line to: `"psr-4": { "Google\\Service\\Generator\\": "src" }` will allow you to have the following structure:

- src/
    - Method.php
    - Resource.php
    - ...

Reply:
☑️